pkg/sql/catalog/schematelemetry/schematelemetrycontroller/testdata/cron_rewrites

```diff
@@ -0,0 +1,53 @@
+test
+440aaea9-06f5-43c1-94d4-fb2b57601ebd
+f715f667-f9a0-436e-90ca-723cee42892c
+15698605-3f40-452c-8ab4-d3a46b27d22b
+5b0ae8e3-9d4b-40a2-960f-6dd3b6df248f
+576451b4-3ff4-485f-aa7b-11e274c1fe21
+df6d7374-6530-4a82-b2f8-974f2ecd8d87
+16240126-6dc5-4bb3-946c-49509e49167f
+6d203888-666e-41ce-be9a-66e4ac3cac01
+cbb622ba-6aad-4088-b212-f1b004985b7d
+e43be3e5-ef0d-41a5-934c-ac25fe9968f2
+f6217ae6-9e67-43f2-9b11-02eca38570f4
+25990464-c11d-415c-88a4-da1e87485770
+bd038643-7671-4813-be1e-0c3044772e08
+00000000-0000-0000-0000-000000000000
+00000001-0001-0001-0001-000000000001
+81cfb104-3ea8-11ed-b878-0242ac120002
+81cfb488-3ea8-11ed-b878-0242ac120002
+81cfb582-3ea8-11ed-b878-0242ac120002
+81cfb672-3ea8-11ed-b878-0242ac120002
+81cfb762-3ea8-11ed-b878-0242ac120002
+81cfb83e-3ea8-11ed-b878-0242ac120002
+81cfb92e-3ea8-11ed-b878-0242ac120002
+81cfbc4e-3ea8-11ed-b878-0242ac120002
+81cfbd48-3ea8-11ed-b878-0242ac120002
+81cfbe24-3ea8-11ed-b878-0242ac120002
+----
+                                      @hourly     @daily       @weekly
+440aaea9-06f5-43c1-94d4-fb2b57601ebd  22 * * * *  22 10 * * *  22 10 * * 5
```

Comment:
for `@weekly` why do we distribute on `month`? shouldn't we distribute on `day` instead?

Reply:
https://crontab.guru/#22_10_*_*_5

I don't think I distribute on month. The last column IIUC, is day of the week.

Reply:
heh, I think I looked at some bad oracle doc...